erdgo/cmd/testTransaction/main.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-sdk/erdgo"
+)
+
+func main() {
+	ep := erdgo.NewElrondProxy("http://174.138.103.62:8079")
+
+	// Load a wallet .PEM file
+	privateKey, err := erdgo.LoadPrivateKeyFromPemFile("../../tests/alice.pem")
+	if err != nil {
+		fmt.Printf("Unable to load alice.pem: %s\n\r", err)
+		return
+	}
+	// Generate address from private key
+	address, err := erdgo.GetAddressFromPrivateKey(privateKey)
+	if err != nil {
+		fmt.Printf("Error generating address: %s\n\r", err)
+		return
+	}
+	// Get account info
+	account, err := ep.GetAccount(address)
+	if err != nil {
+		fmt.Printf("Error retrieving account info: %s\n\r", err)
+		return
+	}
+	// Get network configuration
+	networkConfig, err := ep.GetNetworkConfig()
+	if err != nil {
+		fmt.Printf("Error retrieving network config: %s\n\r", err)
+		return
+	}
+
+	// Create a transaction
+	tx := &erdgo.Transaction{
+		ChainID:  networkConfig.ChainID,
+		Version:  networkConfig.MinTransactionVersion,
+		GasLimit: networkConfig.MinGasLimit,
+		GasPrice: networkConfig.MinGasPrice,
+		Nonce:    account.Nonce,
+		SndAddr:  address,
+		RcvAddr:  address,
```

Comment:
alice to alice tx?

Reply:
It's just an example application that demonstrates the use of transactions' creating, signing, broadcasting and getting back the result.